src/contracts/implementations/arbitrable/MultipleArbitrable.js

```diff
@@ -1,6 +1,5 @@
 import _ from 'lodash'
```

Comment:
This file shouldn't exist. The contract should be made standard compliant and then just use Arbitrable.js to fetch evidence and MetaEvidence